src/modules/sync/sync/assetFolders.ts

```diff
@@ -15,12 +15,13 @@ export const syncAssetFolders = async (
   operations: DiffModel["assetFolders"],
   logOptions: LogOptions,
 ) => {
-  logInfo(logOptions, "standard", "Updating asset folders");
-
   if (!operations.length) {
+    logInfo(logOptions, "standard", "No asset folders to update");
```

Comment:
Just to avoid confusion with other entities that have `add/update/delete`, I would use a different wording here. e.g. "No asset folders updates"